worker.py

```diff
@@ -0,0 +1,50 @@
+from yapapi import WorkContext
+import json
+
+
+def parse_result(raw_data):
```

Comment:
A docstring with example input and out would help greatly to understand what this method does. Or, even better, a unit test.

Reply:
The `yapapi` interface changed between the day this PR was created and today (4214f34).
Now it is simpler, and also it will change when the non-mocked runtime.